### DISCUSSION TITLE
SIGSEGV on simulateContract

### DISCUSSION THREAD

**Discussion by gershido** [wevm/viem#742] [Question]
I'm using Anvil to fork Mainnet and run tests. Using `simulateContract`, in case the simulation should revert, the program collapses with a SIGSEGV error. Is there a known issue? If not, i'll try to make a reproducible package. 
Thank you!

**Comment by jxom** [comment]
Hey! Our test suite also runs on a forked mainnet using Anvil. I wonder what would be causing a SIGSEGV error though. A reproduction would help a lot!

**Reply by gershido** [reply]
Does it reproduce on your side?

**Comment by gershido** [comment]
Reproduction here:
https://github.com/gershido/viem-sigsegv
